### ISSUE TITLE
Training loss vs. Validation loss on Wandb

### ISSUE THREAD

**Issue by Tongcheng** [tloen/alpaca-lora#534]
I have a question about not sure how to show both the training loss and the validation set loss during training in wandb, by default, what I see in wandb is training loss (a curve), and also there's a single point of "train / train loss". 
![image](https://github.com/tloen/alpaca-lora/assets/6565047/e90ce5f4-6e6e-45c4-b5b2-2fdca523e3f6)
![image](https://github.com/tloen/alpaca-lora/assets/6565047/2f8750d9-fc25-4fd9-9e99-fea6318a6e4a)

Do you know if train/loss represents the training set loss function over time, or is it validation set? How to add the other?

**Comment by freQuensy23-coder** [comment]
Same q
